durabletask-azurefunctions/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For more information on pyproject.toml, see https://peps.python.org/pep-0621/
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "durabletask.azurefunctions"
+version = "0.1.0"
```

Comment:
Victoria - what versioning strategy would you propose if the first version that goes to PyPi would be used for internal testing only?